doc/features.rst

```diff
@@ -17,7 +17,7 @@ ________
   
 * Supports a truly reproducible workflow by minimizing the code needed for analysis and visualization.
 * Already used in a variety of research projects, from conception to final publication.
-* All HoloViews objects can be pickled and unpickled.
+* All HoloViews objects can be pickled and unpickled, with no plotting-library dependencies.
```

Comment:
I agree that there should never be a case where you can't unpickle the data itself just because a plotting library is missing.

What I wanted to check is that styles (mostly) have the same behavior: although technically styles *aren't* part of the  holoviews elements (so what you say above is true if you replace the word 'object' with 'elements') a user might also expect styles to pickle/unpickle in a way that is agnostic to the plotting library.